pkg/analyzer/analyzers/asana/asana.go

```diff
@@ -102,3 +148,50 @@ func printMetadata(me *MeJSON) {
 	}
 	t.Render()
 }
+
+func allPermissions() []analyzers.Permission {
+	asanaPermissions := []Permission{
+		AllocationManagementRead,
+		AllocationManagementWrite,
+		AttachmentManagementRead,
+		AttachmentManagementWrite,
+		AuditLogRead,
+		AuditLogWrite,
+		BatchApiRead,
+		BatchApiWrite,
+		EventsRead,
+		EventsWrite,
+		GoalsRead,
+		GoalsWrite,
+		JobsRead,
+		JobsWrite,
+		PortfolioRead,
+		PortfolioWrite,
+		ProjectRead,
+		ProjectWrite,
+		TagsRead,
+		TagsWrite,
+		TaskRead,
+		TaskWrite,
+		TeamRead,
+		TeamWrite,
+		UserRead,
+		UserWrite,
+		UserTaskRead,
+		UserTaskWrite,
+		MembershipRead,
+		MembershipWrite,
+		RuleRead,
+		RuleWrite,
+	}
```

Comment:
nit: you can just iterate one of the generated Permission maps.

Reply:
Does this access depend on the permissions of the user that generated the token?

Reply:
> Does this access depend on the permissions of the user that generated the token?

In Asana, Personal Access Tokens (PATs) do not have specific, customizable permissions. The permissions of a PAT are inherently tied to the permissions of the Asana user account that creates the token